schemas/+aod/trackMotion3D.m

```diff
@@ -0,0 +1,86 @@
+
+function [x,y,z,t] = trackMotion3D(fn, analysischan) 
+
+%% test function computeshifts
+% requires z:\libraries\matlab in path
+run(getLocalPath('/lab/libraries/hdf5matlab/setPath.m')) ;
```

Comment:
this shouldn't be here. the user should have set up the path correctly before hand. this would likely have unintended side effects.